Dockerfile

```diff
@@ -43,8 +45,8 @@ RUN DEBIAN_FRONTEND=noninteractive apt install libnss-ldapd -y
 RUN sed -i 's/files/ldap files/g' /etc/nsswitch.conf
 
 # Make editable and debuggable
-RUN pip install debugpy
-RUN pip install -e .
+RUN uv tool install debugpy
```

Comment:
Ah, this is why you need uv tools to be on your path, but I don't see why you need it in the devcontainer.json too?